autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -153,6 +171,33 @@ function sortSections(sections: SectionSelected[]): SectionSelected[] {
   return [].concat(...sectionsForProfs.values(), ...TBASections);
 }
 
+/**
+ * Fetches sections for course in courseCard, then updates courseCard with new sections
+ * @param courseCard course to get sections for
+ * @param term term to fetch sections for
+ */
+async function createCourseCardFrom(
```

Comment:
I would change the name of this to mention that it's doing a fetch (e.g. add a `fetch` or `retrieve` to the name)